lib/membrane/video_compositor/compound/layout.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Membrane.VideoCompositor.Compound.Layout do
+  @moduledoc """
+  Behaviour representing layous - modification combing multiple objects into one.
+
+  Render callback is a placeholder.
+  """
+
+  @type definition_t :: struct() | module()
```

Comment:
I thought that in Elixir we usually use:
```suggestion
  @type t :: struct() | module()
```

Reply:
ok